src/win/pty.cc

```diff
@@ -126,6 +126,15 @@ static std::wstring get_shell_path(std::wstring filename)  {
   return shellpath;
 }
 
+void throw_winpty_error(const char *generalMsg, winpty_error_ptr_t error_ptr) {
+  std::stringstream why;
+  std::wstring msg(winpty_error_msg(error_ptr));
+  std::string msg_(msg.begin(), msg.end());
```

Comment:
Thanks for the feedback, tracking in https://github.com/Tyriar/node-pty/issues/104 🙂